mclagsyncd/mclag.h

```diff
@@ -23,13 +23,32 @@
 #define MCLAG_DEFAULT_IP   0x7f000006
 
 enum MCLAG_FDB_OP_TYPE {
-    MCLAG_FDB_OPER_ADD =1,
+    MCLAG_FDB_OPER_ADD = 1,
     MCLAG_FDB_OPER_DEL = 2,
 };
 
 enum MCLAG_FDB_TYPE {
-    MCLAG_FDB_TYPE_STATIC = 1,
+    MCLAG_FDB_TYPE_STATIC  = 1,
     MCLAG_FDB_TYPE_DYNAMIC = 2,
+    MCLAG_FDB_TYPE_DYNAMIC_LOCAL = 3, /* Used while sending MAC to Syncd to program with aging enabled. */
+};
+
+enum MCLAG_DOMAIN_CFG_OP_TYPE {
+    MCLAG_CFG_OPER_NONE       = 0,  //NOP
+    MCLAG_CFG_OPER_ADD        = 1,  //Add domain
+    MCLAG_CFG_OPER_DEL        = 2,  //Delete domain
+    MCLAG_CFG_OPER_UPDATE     = 3,   //update domain
+    MCLAG_CFG_OPER_ATTR_DEL   = 4   //Attribute del 
+};
+
+
```

Comment:
Is this extra empty line needed? If not, could you please remove this one and others throughout your changes?